File: CineCodeV2/Domain/Entities/Filme.cs
namespace CineCodeV2.Domain.Entities;

public class Filme
{
    public int Id { get; set; }
    public string Titulo { get; set; } = string.Empty;
    public DateOnly AnoLancamento { get; set; }
    public string Autor { get; set; } = string.Empty;
    public TimeOnly Duracao { get; set; }
    public int StatusFilme { get; set; }
    public string Sinopse { get; set; } = string.Empty;
    public int Classificacao { get; set; }
    public string Imagem { get; set; } = string.Empty;

}

File: CineCodeSLS/src/app/models/filme.ts
import { Time } from "@angular/common";

export interface Filme {
    id: number;
    titulo: string;
    anoLancamento: Date; 
    autor: string;
    duracao: Time;   
    statusFilme: number;
    sinopse: string;
    classificacao: number;
    imagem: string;
  }
  